core/src/main/java/io/questdb/griffin/engine/functions/groupby/AbstractCovarGroupByFunction.java

```diff
@@ -134,16 +140,23 @@ protected void aggregate(MapValue mapValue, double x, double y) {
         double meanY = mapValue.getDouble(valueIndex + 1);
         double sumXY = mapValue.getDouble(valueIndex + 2);
         long count = mapValue.getLong(valueIndex + 3) + 1;
+        double sumX = mapValue.getDouble(valueIndex + 4);
```

Comment:
this will regress performance of the functions that do not use the extra values. In this case it is better to break the inheritance and create a standalone slope function.